mlflow/utils/databricks_utils.py

```diff
@@ -418,8 +423,23 @@ def _fail_malformed_databricks_auth(profile):
         "https://github.com/databricks/databricks-cli." % profile
     )
 
+# Helper function to attempt to read OAuth Token from
+# mounted file in Databricks Model Serving environment
+def _get_model_dependency_oauth_token(should_retry=True):
+    try:
+        with open(MODEL_DEPENDENCY_OAUTH_TOKEN_FILE_PATH, "r") as f:
```

Comment:
is revelaing this going to be a security concern at all?

Reply:
if we don't want to do that we may need to bake into vector search SDK which is proprietary